Description: Fix errors in sntp man page
 W: ntp: manpage-has-errors-from-man usr/share/man/man1/sntp.1.gz 44: warning: macro `1996-10-15' not defined
 I: ntp: hyphen-used-as-minus-sign usr/share/man/man1/sntp.1.gz:133
Author: Peter Eisentraut <petere@debian.org>

--- a/sntp/sntp.1
+++ b/sntp/sntp.1
@@ -40,8 +40,7 @@ and the
 .B "'+4.567 +/- 0.089 secs'"
 indicates the local clock is 4.567 seconds behind the correct time
 (so 4.567 seconds must be added to the local clock to get it to be correct),
-and the time of
-'1996-10-15 20:17:25.123'
+and the time of '1996-10-15 20:17:25.123'
 is believed to be correct to within
 +/- 0.089
 seconds.
@@ -130,7 +129,7 @@ This option takes an integer number as i
 .sp
 This option enables authentication using the key specified in this option's argument.
 The argument of this option is the keyid, a number specified in the keyfile as this
-key's identifier. See the keyfile option (-k) for more details.
+key's identifier. See the keyfile option (\-k) for more details.
 .TP
 .BR \-k " \fIfile-name\fP, " \--keyfile "=" \fIfile-name\fP
 Specify a keyfile. SNTP will look in this file for the key specified with \-a.
